workers/docker-worker/test/integration/file_artifact_test.js

```diff
@@ -273,7 +273,8 @@ helper.secrets.mockSuite(suiteName(), ['docker', 'ci-creds'], function(mock, ski
     assert.equal(result.artifacts['public/my-missing.txt'].storageType, 'error');
   });
 
-  test('upload retry', async () => {
+  // intermittent https://github.com/taskcluster/taskcluster/issues/2951
+  test.skip('upload retry', async () => {
```

Comment:
It isn't clear to me if the test is faulty, or it has found a genuine issue. Looking at the test, I can imagine it could be intermittently blocking itself through some side effect of the synchronisation mechanics, but I don't have expertise in synchronisation in node.js, so I'm not well placed to make a judgement.

If it isn't immediately obvious what the problem is, I suspect it is reasonable to disable the test. Although it doesn't give me a good feeling in general, especially if it is going to be disabled for a while. However, I also suspect the retry mechanics of uploads don't see a lot of maintenance, so there is a good chance it has always been a flaky test.